pkg/terra/relay.go

```diff
@@ -25,7 +35,13 @@ type Logger interface {
 }
 
 type MsgEnqueuer interface {
-	Enqueue(contractID string, msg []byte) (int64, error)
+	// Enqueue enqueues msg for broadcast and returns its id.
```

Comment:
is there no way to avoid expanding this interface? I was hoping to keep it to actually just the Enqueue method

Reply:
Yeah, could use a separate or extended interface.

Reply:
Updated with separate TxManager, and dropped Start/Close methods.